modules/before-updating-web-console.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-web-console.adoc
+
+:_content-type: CONCEPT
+[id="before-updating-web-console{context}"]
+= Before updating using the web console
+
+Before updating using the web console, you should:
+
+* Have access to the cluster as a user with `cluster-admin` privileges.
+* Have a recent etcd backup in case your update fails and you must restore your cluster to a previous state.
+* You must replace {op-system-base}7 workers with {op-system-base}8 or {op-system} workers before updating to {product-title} {product-version}. Red Hat does not support in-place {op-system-base}7 to {op-system-base}8 updates for {op-system-base} workers; those hosts must be replaced with a clean operating system install.
+* Ensure all Operators previously installed through Operator Lifecycle Manager (OLM) are updated to their latest version in their latest channel. Updating the Operators ensures they have a valid update path when the default OperatorHub catalogs switch from the current minor version to the next during a cluster update.
+* Ensure that all machine config pools (MCPs) are running and not paused. Nodes associated with a paused MCP are skipped during the update process. You can pause the MCPs if you are performing a canary rollout update strategy.
+//remove this???^ or maybe just add another bullet that you can break up the update?
+* To accommodate the time it takes to update, you are able to do a partial update by updating the worker or custom pool nodes. You can pause and resume within the progress bar of each pool.
+* If your cluster uses manually maintained credentials, update the cloud provider resources for the new release.
+* Review the list of APIs that were removed in Kubernetes 1.26, migrate any affected components to use the new API version, and provide the administrator acknowledgment.
+
+
+//may need to move to a troubleshooting section
+//* If you run an Operator or you have configured any application with the pod disruption budget, you might experience an interruption during the update process. If `minAvailable` is set to 1 in `PodDisruptionBudget`, the nodes are drained to apply pending machine configs which might block the eviction process. If several nodes are rebooted, all the pods might run on only one node, and the `PodDisruptionBudget` field can prevent the node drain.
```

Comment:
If you do end up moving those other prereqs into relevant modules and this module only has a list of "considerations to make", this would be a good item to uncomment and include in the list above